fetools-app/src/components/CopyButton.jsx

```diff
@@ -0,0 +1,19 @@
+function CopyButton({ onCopy }) {
+  const handleCopy = () => {
+    const textToCopy = onCopy();
+    if (navigator.clipboard && textToCopy) {
+      navigator.clipboard.writeText(textToCopy).then(
+        () => {
+          console.log("Text copied to clipboard");
```

Comment:
Once the toast component is merged, we can output this note as a toast. It should include at least the first few words of what was copied.